corehq/apps/change_feed/producer/postgres.py

```diff
@@ -0,0 +1,61 @@
+import select
+import json
+import psycopg2.extensions
+from kafka import KeyedProducer
+from pillowtop.feed.interface import ChangeMeta
+
+from ..data_sources import get_document_store, POSTGRES
+from ..utils import send_to_kafka
+
+CHANNELS = {
```

Comment:
i actually think this should be moved to the `settings.py` file like the pillows are